mParticle-Apple-SDK/MPConstants.swift

```diff
@@ -50,3 +50,115 @@ struct MessageKeys {
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueProductAction = "product_action"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueUnknown = "unknown"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueAddToCart = "add_to_cart"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueRemoveFromCart = "remove_from_cart"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueCheckout = "checkout"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueCheckoutOption = "checkout_option"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueClick = "click"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueViewDetail = "view_detail"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValuePurchase = "purchase"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueRefund = "refund"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueAddToWishlist = "add_to_wishlist"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueRemoveFromWishlist = "remove_from_wish_list"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValuePromotionAction = "promotion_action"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueProductImpressions = "product_impressions"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueCriteria = "criteria"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueAction  = "action"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueImpressionUnknown = "unknown"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueImpressionView = "view"
+    static let kMPRemoteConfigDataPlanningDataPlanVersionValueImpressionClick = "click"
+}
+
+struct ConsentFiltering {
+    static let kMPConsentKitFilter = "crvf"
+    static let kMPConsentKitFilterIncludeOnMatch = "i"
+    static let kMPConsentKitFilterItems = "v"
+    static let kMPConsentKitFilterItemConsented = "c"
+    static let kMPConsentKitFilterItemHash = "h"
+    static let kMPConsentRegulationFilters = "reg"
+    static let kMPConsentPurposeFilters = "pur"
+    static let kMPConsentGDPRRegulationType = "1"
+    static let kMPConsentCCPARegulationType = "2"
+    static let kMPConsentCCPAPurposeName = "data_sale_opt_out"
+}
+
+struct Notifications {
+    static let kMPCrashReportOccurredNotification = "MPCrashReportOccurredNotification"
+    static let kMPConfigureExceptionHandlingNotification = "MPConfigureExceptionHandlingNotification"
+    static let kMPUserNotificationDictionaryKey = "MPUserNotificationDictionaryKey"
+    static let kMPUserNotificationActionKey = "MPUserNotificationActionKey"
+    static let kMPRemoteNotificationDeviceTokenNotification = "MPRemoteNotificationDeviceTokenNotification"
+    static let kMPRemoteNotificationDeviceTokenKey = "MPRemoteNotificationDeviceTokenKey"
+    static let kMPRemoteNotificationOldDeviceTokenKey = "MPRemoteNotificationOldDeviceTokenKey"
+}
```

Comment:
```suggestion
struct Notifications {
    static let kMPCrashReportOccurredNotification = Notification.Name("MPCrashReportOccurredNotification")
    static let kMPConfigureExceptionHandlingNotification = Notification.Name("MPConfigureExceptionHandlingNotification")
    static let kMPUserNotificationDictionaryKey = Notification.Name("MPUserNotificationDictionaryKey")
    static let kMPUserNotificationActionKey = Notification.Name("MPUserNotificationActionKey")
    static let kMPRemoteNotificationDeviceTokenNotification = Notification.Name("MPRemoteNotificationDeviceTokenNotification")
    static let kMPRemoteNotificationDeviceTokenKey = Notification.Name("MPRemoteNotificationDeviceTokenKey")
    static let kMPRemoteNotificationOldDeviceTokenKey = Notification.Name("MPRemoteNotificationOldDeviceTokenKey")
}
```

For Swift notification constants, it's best to create them as Notification types so that they don't need to be converted at the usage sites.